src/lib/components/ui/ArticleList.svelte

```
@@ -1,129 +1,131 @@
+			newArticleRef = document.getElementById(`article-${previousVisibleCount}`);
+		}
+	});
+
+	// Initialize selectedCategory from URL
+	$effect(() => {
+		const highlightParam = $page.url.searchParams.get('highlight');
+		selectedCategory = highlightParam || '';
+	});
+
+	// Update URL when category changes
+	function handleCategoryClick(category: string) {
```

ESLint

[error] 'handleCategoryClick' is defined but never used.

(@typescript-eslint/no-unused-vars)